src/statd/statd.c

```diff
@@ -391,6 +391,31 @@ static int ly_add_ip_link_br(const struct ly_ctx *ctx, struct lyd_node **parent,
 	return SR_ERR_OK;
 }
 
+static int iface_has_flag(json_t *j_iface, const char *ifflag)
+{
+	json_t *flags, *flag;
+	int found = 0;
```

Comment:
I don't see the need for this flag? Why not just return 1 when you find the flag?
Something like:
```
if (strcmp(json_string_value(j_flag), flag) == 0)
    return 1
```